main_test.go

```diff
@@ -330,6 +330,9 @@ func Test_validateBaseURL(t *testing.T) {
 }
 
 func Test_preparedServer(t *testing.T) {
+	// For the purpose of testing - ensure we're using IPv4.
+	conf.BaseURL = "http://127.0.0.1:8000/"
+
```

Comment:
I know we're just doing minimal changes in this PR, so this is _not_ for this change, but I've written the code that does random port allocation so often that I now can't not mention it:

https://go.dev/play/p/GerJqydR1a_F